projects/core/src/cart/facade/cart.service.spec.ts

```diff
@@ -414,4 +410,58 @@ describe('CartService', () => {
       expect(result).toEqual(cart);
     });
   });
+
+  describe('getAppliedVouchers', () => {
+    it('should return voucher array', () => {
+      const testCart: Cart = <Cart>{
+        appliedVouchers: [{ code: 'voucher1' }, { code: 'voucher2' }],
+      };
+      store.dispatch(new fromCart.LoadCartSuccess(testCart));
+
+      let result: Voucher[];
+      service
+        .getAppliedVouchers()
+        .subscribe(value => (result = value))
```

Comment:
You can expect result in subscribe section
.subscribe(value => (expect(value).toEqual(testCart.appliedVouchers)))

Reply:
updated